bellows/ezsp/v4/__init__.py

```diff
@@ -193,3 +194,41 @@ async def read_counters(self) -> dict[t.EmberCounterType, t.uint16_t]:
     async def read_and_clear_counters(self) -> dict[t.EmberCounterType, t.uint16_t]:
         (res,) = await self.readAndClearCounters()
         return dict(zip(t.EmberCounterType, res))
+
+    async def set_extended_timeout(
+        self, nwk: t.NWK, ieee: t.EUI64, extended_timeout: bool = True
+    ) -> None:
+        (curr_extended_timeout,) = await self.getExtendedTimeout(remoteEui64=ieee)
+
+        if curr_extended_timeout == extended_timeout:
+            return
+
+        (node_id,) = await self.lookupNodeIdByEui64(eui64=ieee)
+
+        # Check to see if we have an address table entry
+        if node_id != 0xFFFF:
+            await self.setExtendedTimeout(
+                remoteEui64=ieee, extendedTimeout=extended_timeout
+            )
+            return
+
+        (status, addr_table_size) = await self.getConfigurationValue(
```

Comment:
Can we capture this when applying the config so it is only done once when the application starts?